functionary/ui/tables/schedule_task.py

```diff
@@ -19,21 +27,18 @@ class ScheduledTaskTable(tables.Table):
     last_run = tables.DateTimeColumn(
         accessor="most_recent_task__created_at",
         verbose_name="Last Run",
-        linkify=lambda record: reverse(
-            "ui:task-detail", kwargs={"pk": record.most_recent_task}
-        ),
-        format="N j, Y, g:i a",
+        linkify=lambda record: generateLastRunUrl(record),
```

Comment:
Minor thing that doesn't really need to change, but I'll point out that this could have just been `linkify=generateLasRunUrl`, since `linkify` takes a callable.